kubernetes/config.aat.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${SERVICE_NAME}-config
+  namespace: ${NAMESPACE}
+data:
+  IDAM_API_BASE_URI: "https://preprod-idamapi.reform.hmcts.net:3511" #TODO Are these correct? Written by Alec
+  S2S_BASE_URI: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+# TODO This data is NOT referenced in any sections within deployment.template.yaml. Can be added by the following lines, should it be?
```

Comment:
correct needs referencing

Reply:
@timja which parts of deployment.template.yaml need this to be referenced?

Reply:
https://github.com/hmcts/bulk-scan-processor/blob/master/kubernetes/deployment.template.yaml#L99-L100